src/ddmd/traits.d

```diff
@@ -403,6 +404,47 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
         return new IntegerExp(e.loc, false, Type.tbool);
     }
 
+    /**
+     if traitsFuncArg is passed either a delegate. a function
+     or a function pointer or a variable of types.
+     it will return the functionType.
+     else it will return null
+
+     In case of a function declaration being passed verbatim
+     it will set the second  optional argument to the FuncDeclarartion
+     given it was provided.
+    */
```

Comment:
Please use Ddoc style documentation. It this case it means having a `Params:` section and a `Returns:` section. This is the standard for new functions.